### PR TITLE
feat: ssr catalogue landing page

### PR DESCRIPTION
**Pull request by connoratrug**
Add landing page in front of cohorts listing 

- Add landing page
- Use landing page details from schema settings
- Improve styling for mobile vue 
- Implement pixelpillow icon pipeline for new icons 

See readme and docs for details

### CODE REVIEW

File: apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/cohorts/index.vue [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+]);
+
+let search = computed(() => {
+  // @ts-ignore
+  return filters.find((f) => f.columnType === "_SEARCH").search;
+});
+
+const query = computed(() => {
+  return `
+  query Cohorts($filter:CohortsFilter, $orderby:Cohortsorderby){
```
Comment (mswertz) [inline apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/cohorts/index.vue:64]: don't you want gql autoformat?

Comment (connoratrug) [reply apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/cohorts/index.vue:64]: yea, it started out as really small request , single line , i'll add the gql

Comment (connoratrug) [reply apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/cohorts/index.vue:64]: added

**Review by mswertz — APPROVED** [review]

File: apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/cohorts/index.vue (hunk 1 above)
Comment (connoratrug) [reply apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/cohorts/index.vue:64]: and reverted, has issue with nuxt fetch , need to look into with this does not work in this context

**Pull request merged**